WPMedia/fusion-news-theme-blocks
# Tmedia 435 logged in cta

## Description
Shows logged in info 

## Jira Ticket
- [TMEDIA-435](https://arcpublishing.atlassian.net/browse/TMEDIA-435)

## Acceptance Criteria
Acceptance Criteria

Desktop navigation displays “Account” button when logged in

Mobile navigation displays “Account” button when logged in

On click, dropdown menu appears with Manage account and Sign out links

## Test Steps
1. Checkout this branch `git checkout TMEDIA-435-logged-in-cta`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block,@wpmedia/shared-styles`
3. Change in the file of the project to see the user set to true
![Screen Shot 2021-09-09 at 10 06 18](https://user-images.githubusercontent.com/5950956/132711851-62ed1414-20bd-46a0-a31c-282a4ea5b01c.png)
4. See the chevron shown up and down on the right after having the user fake logged in 

## Effect Of Changes
### Before
- no logged in state 

### After
- can see logged in state on 

## Dependencies or Side Effects
- added dual icon button type 

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
